packages/components/src/atoms/List/List.tsx

```diff
@@ -17,10 +17,8 @@ interface BaseProps {
   marker?: boolean
 }
 
-export type ListProps<T extends ElementType> = PolymorphicComponentPropsWithRef<
-  T,
-  BaseProps
->
+export type ListProps<T extends ElementType = 'ul'> =
```

Comment:
is it related to `LinkButton`?

Reply:
It was to quick fix the `nextjs.store` build